content_script.js

```diff
@@ -7,6 +7,11 @@ var imageFormatExtension = "jpeg";
 
 // Take screenshot
 captureScreenshot = function () {
+  // Check for encrypted content
+  if (document.querySelector('video').mediaKeys != null) {
```

Comment:
`document.querySelector("video")` is already queried several lines after and store in `video` variable. Please use it to avoid querying twice the node.